cmake_modules/FindCuVSLAM.cmake

```diff
@@ -4,6 +4,7 @@
 #
 # It sets the following variables:
 #  CUVSLAM_FOUND         - Set to false, or undefined, if cuVSLAM isn't found.
+#  CuVSLAM_VERSION       - The major version of cuVSLAM found (e.g., "14").
```

Comment:
We should expect format like `14.0.0`

The minor version seems available: https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_nitros/blob/8d2a14f301f8b169626637c8d87feb8317e22369/isaac_ros_nitros/lib/cuvslam/include/cuvslam.h#L40-L44

keep 0 for the patch.